packages/slice-machine/components/AppLayout/Navigation/ChangesRightElement.tsx

```diff
@@ -0,0 +1,33 @@
+import { FC } from "react";
+
+import { useUnSyncChanges } from "@src/hooks/useUnSyncChanges";
+import { RightElement } from "@src/components/SideNav/SideNav";
+import { AuthStatus } from "@src/modules/userContext/types";
+import { useNetwork } from "@src/hooks/useNetwork";
+
+export const ChangesRightElement: FC = () => {
+  const isOnline = useNetwork();
```

Comment:
What is the best for you, I used it from there at the beginning but since it's coming from useNetwork, I kept it like this.
When we will rework useUnsyncedChanges it could simplify the hook. Each hook should have separate concern and isOnline don't look too much related. WDYT?

Reply:
Is there rework to be done on useUnsyncedChanges?.... that may change things

Reply:
Yes for me, we should try to not mix everything, there is no direct link between useUnsyncedChanges and useNetwork. It create a bit of a all in one hook.

Reply:
But we will see later on :)